internal/hud/webview/convert.go

```diff
@@ -327,17 +370,18 @@ func TiltfileResourceProtoView(name model.ManifestName, ms *store.ManifestState,
 	} else {
 		tr.Status.LastDeployTime = finish
 	}
+
+	tr.Status.Conditions = []v1alpha1.UIResourceCondition{
+		UIResourceReadyCondition(tr.Status),
+	}
+
 	return tr
 }
 
 func populateResourceInfoView(mt *store.ManifestTarget, r *v1alpha1.UIResource) error {
 	r.Status.UpdateStatus = mt.UpdateStatus()
 	r.Status.RuntimeStatus = v1alpha1.RuntimeStatusNotApplicable
 
-	if mt.Manifest.PodReadinessMode() == model.PodReadinessIgnore {
-		return nil
-	}
```

Comment:
Do we need any extra logic in the K8s case below to handle this correctly?

(Previously it looks like `K8sResourceInfo` would be nil in this case but will now get populated, not sure of impact of reporting on the Pod status when we're ignoring its readiness will be, if any)

Reply:
ya, i think this behavior was wrong, and is leftover from when RuntimeStatus didn't properly handle PodReadinessMode. and i think we do want K8sResourceInfo to get populated with the resources we deployed!